codalab/lib/bundle_cli.py

```diff
@@ -1899,6 +1899,10 @@ def recurse(old_bundle_uuid):
                         new_metadata['name'] = new_output_name + '-' + \
                                                old_info['metadata']['name']
 
+                # Put docker_image in requested_docker_image if it is present and this is a run bundle
```

Comment:
Document more based on our discussion: say that by default, we copy request_docker_image, but this is inappropriate because tags can point to different digest (give an example).